src/components/Checkout/LoanReservation.vue

```diff
@@ -89,38 +99,42 @@ export default {
 		},
 		reservationMessage() {
 			if (this.expiryTime !== null) {
-				const reservedDate = new Date(this.expiryTime);
-				const mins = differenceInMinutes(reservedDate.getTime(), Date.now());
-				const seconds = differenceInSeconds(reservedDate.getTime(), Date.now()) % 60;
+				const mins = differenceInMinutes(this.reservedDate.getTime(), Date.now());
+				const seconds = differenceInSeconds(this.reservedDate.getTime(), Date.now()) % 60;
 
-				let warningMessageUpperBoundMinutes = 6;
+				let warningMessageUpperBoundMinutes = 5;
 				let differenceInWords = `Reserved for ${mins} more minutes`;
 
 				if (this.activateTimer === true) {
-					warningMessageUpperBoundMinutes = 10;
+					warningMessageUpperBoundMinutes = 9;
 					differenceInWords = `Reservation expires in ${mins}m ${seconds}s`;
 				}
 
-				if ((reservedDate.getTime() - Date.now()) <= 0 || this.isExpiringSoon) {
+				if ((this.reservedDate.getTime() - Date.now()) <= 0 || this.isExpiringSoon) {
 					clearInterval(this.reservationMessageId);
 					this.loanReservationMsg1 = true;
 					this.loanReservationMsg2 = false;
 					this.loanReservationMsg3 = false;
 					this.loanReservationMsg4 = false;
 				} else if (mins > warningMessageUpperBoundMinutes) {
 					this.setDifferenceInWords(differenceInWords);
+					this.loanReservationMsg1 = false;
```

Comment:
reassign prior messages to false... goes for those below too. This along with the markup refactor prevents multiple loan reservation statements from showing up.